dev/Breadcrumb/InteractionTests/BreadcrumbTests.cs

```diff
@@ -43,9 +44,218 @@ public void TestCleanup()
         }
 
         [TestMethod]
-        public void BasicTest()
+        [TestProperty("TestSuite", "A")]
+        public void NoInteractionTest()
+        {
+            using (var setup = new TestSetupHelper("Breadcrumb Tests"))
+            {
+                Log.Comment("Retrieve breadcrumb control as generic UIElement");
+                UIObject breadcrumb = FindElement.ByName("BreadcrumbControl");
+                Verify.IsNotNull(breadcrumb, "Verifying that we found a UIElement called BreadcrumbControl");
+
```

Comment:
Finding BreadCrumbControl is the first thing you do in every test. You can extract these two lines into a method in order not to repeat them every time.